src/utils.h

```diff
@@ -51,6 +56,11 @@ typedef enum {I, O, T, Z, S, J, L} BlockType;
 typedef enum {LEFT, NO_ROTATE, RIGHT} Rotation;
 
 /*
+ * Modes that the game can be in
+ */
+typedef enum {RUNNING, PAUSED, SHUTDOWN, BOSS} Gamemode;
```

Comment:
can you change this and the `Rotation` enum above to not be all caps, since those should be reserved for constants